packages/features/ee/dsync/lib/users/dSyncUserSelect.ts

```diff
@@ -0,0 +1,17 @@
+const dSyncUserSelect = {
```

Comment:
This constant is inferred as any

Reply:
This is showing up as the right type for me.

![image](https://github.com/calcom/cal.com/assets/65426560/2971ab1c-4f49-4ff4-9558-29488718fc50)

Reply:
Looks valid to me @zomars - may be a ts fluke on your end